cmd/lifecycle/cli/command.go

```diff
@@ -24,7 +24,7 @@ type Command interface {
 	Exec() error
 }
 
-func Run(c Command, asSubcommand bool) {
+func Run(c Command, withName string, asSubcommand bool) {
```

Comment:
just a nit.- `withPhaseName` or `phaseName` would be more clear to me than `withName`, but maybe it isn't always the name of a phase?

Reply:
Updated in [389cc1b](https://github.com/buildpacks/lifecycle/pull/1106/commits/389cc1be4409c9790832856eb91d462d2da29dbe)